src/paint.rs

```diff
@@ -517,7 +577,10 @@ impl Paint {
     /// path.circle(50.0, 50.0, 20.0);
     /// canvas.fill_path(&mut path, &bg);
     /// ```
-    pub fn radial_gradient_stops(cx: f32, cy: f32, in_radius: f32, out_radius: f32, stops: &[(f32, Color)]) -> Self {
+    pub fn radial_gradient_stops<Stops>(cx: f32, cy: f32, in_radius: f32, out_radius: f32, stops: Stops) -> Self
```

Comment:
Maybe use impl Trait instead of generics here (`stops: impl IntoIterator<Item = (f32, Color)>`)

Reply:
On second thought, I think I'll go back to the previous source compatible API for now. The limitation of stops is more important IMO.

Reply:
👍